docs/DeveloperGuide.md

```diff
@@ -294,6 +294,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is `Jobby` and the **Actor** is the `user`, unless specified otherwise)
 
+
+
+**Use case: Edit a contact**
+
+**MSS**
+
+1.  User requests to edit a contact
+2.  Jobby edits the contact
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The given request does not match with any contact.
+    * 1a1. Jobby shows an error message.
+    * 
+      Use case ends.
+
+* 2a. The contact file is corrupted.
```

Comment:
Would this part be necessary? I think it is better handled by another use case

Reply:
This has been resolved.